proto/cosmos/protocolpool/v1/query.proto

```diff
@@ -27,3 +32,17 @@ message QueryCommunityPoolResponse {
   repeated cosmos.base.v1beta1.DecCoin pool = 1
       [(gogoproto.castrepeated) = "github.com/cosmos/cosmos-sdk/types.DecCoins", (gogoproto.nullable) = false];
 }
+
+// QueryUnclaimedBudgetRequest is the request type for the Query/UnclaimedBudgetRequest
+// RPC method.
+message QueryUnclaimedBudgetRequest {
+  // address is the recipient address to query unclaimed budget amount for.
+  string address = 1 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+}
+
+// QueryUnclaimedBudgetResponse is the response type for the Query/UnclaimedBudget
+// RPC method.
+message QueryUnclaimedBudgetResponse {
```

Comment:
I think I full view on the budged alloction would be more helpful than only the unclaimed amount.